modules/admin/src/resources/js/directives.js

```diff
@@ -659,6 +659,7 @@
             scope: {
                 "model": "=",
                 "options": "=",
+                "dateFormat": "=",
```

Comment:
I just fixed this problem, but it still won't work as of the validation. https://github.com/luyadev/luya/commit/0b5e79d811199372e8ca186a137ffe2f191eaa21